website/content/docs/platform/servicenow/installation.mdx

```diff
@@ -0,0 +1,91 @@
+---
+layout: docs
+page_title: Vault Credential Resolver Installation
+description: Installation steps for the ServiceNow Vault Credential Resolver.
+---
+
+# Installing the Vault Credential Resolver
+
+## Prerequisites
+
+* ServiceNow version Quebec+ (untested on previous versions)
+* MID server version Quebec+ (untested on previous versions)
+* Discovery and external credential plugins activated on ServiceNow
+* Working Vault deployment accessible from the MID server
+
+## Installing Vault Agent
+
+* Select your desired auth method from Agent's [supported auth methods](/docs/agent/autoauth/methods)
+  and set it up in Vault
+  * For example, to set up AppRole auth and a role called `role1` with the `demo` policy attached:
+
+    ```bash
+    vault auth enable approle
+    vault write auth/approle/role/role1 bind_secret_id=true token_policies=demo
+    ```
+
+  * To get the values required for files `/vault/config/roleID` and
+    `/vault/config/secretID` referenced in the config below, you could then run:
+
+    ```bash
+    roleID=$(vault read -format json auth/approle/role/role1/role-id | jq -r '.data.role_id')
```

Comment:
I am not sure where these variables are being used. I guess they somehow go into the `agent.hcl`?

Reply:
Good question, I've addressed this in 5759c6789dbfb65b282cd65e7cd9bf98e5bd3e9c. LMK if you think it can be improved further